### PR TITLE
Add DBaaS Pukki documentation

### PR DESCRIPTION
**Pull request by oscarkraemer**
https://csc-guide-preview.rahtiapp.fi/origin/DBAAS-61/cloud/dbaas/

The services have received external users so it would be a good idea 
to merge the DBaaS documentation to master.

The service is still in "closed-beta" but it would still be beneficial if the 
documentation would be visible for everybody, since it also works as a
away to build awareness of the service.

### CODE REVIEW

File: docs/cloud/dbaas/advanced.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 !!! error "Closed Beta"
     Pukki DBaaS is in closed beta. This means that services is probably not suitable for most users
     and there might be breaking changes. If you are still interested in using the service you can
-    contact service-desk to see if the service would be suitable for you.
+    contact [service-desk](mailto:servicedesk@csc.fi) to see if the service would be suitable for you.
```
Comment (joonas-somero) [inline docs/cloud/dbaas/advanced.md:5]:
```suggestion
    [contact us](../../support/contact.md) to see if the service would be suitable for you.
```

File: docs/cloud/dbaas/web-interface.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+You can log into DBaaS by going to [pukki.dbaas.csc.fi](https://pukki.dbaas.csc.fi).
+
+## Creating a new database instance
+
+In the column on the left hand side you can go to `Database` -> `Instances` and then press on the `Launch Instance` that can be found on your right hand side.
+Now you can define what settings you want for your database:
+
+1. `Instance name` - What ever you want to name the instance.
+2. `Volume size` - How much disk space you will need for your database. If you just want to test the DBaaS then 1 GiB is probably enough. To increase the disk size later, downtime is require for the database. If you already know how much data you will use then it is easy to estimate how large volume you need.
```
Comment (lvarin) [inline docs/cloud/dbaas/web-interface.md:16]:
```suggestion
2. `Volume size` - How much disk space you will need for your database. If you just want to test the DBaaS then 1 GiB is probably enough. To increase the disk size later, downtime is required for the database. If you already know how much data you will use then it is easy to estimate how large volume you need. 
```

typo

File: docs/cloud/dbaas/web-interface.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Now you can define what settings you want for your database:
+
+1. `Instance name` - What ever you want to name the instance.
+2. `Volume size` - How much disk space you will need for your database. If you just want to test the DBaaS then 1 GiB is probably enough. To increase the disk size later, downtime is require for the database. If you already know how much data you will use then it is easy to estimate how large volume you need. 
+3. `Volume type` - This can be left empty.
+4. `Datastore` - What type of database you want. It is recommend to use the latest version of your preferred database, if you don't have a specific reason for using an older version. PostgreSQL is the only available database at the moment.
+5. `Flavor` - How large database instances you want. For small use cases the `standard.small` is probably enough. If you later find out that it is not large enough you can always change it later. Changing flavor will require downtime.
+6. `Locality` - Is not needed. In the future DBaaS will support clustered databases and at that point anti-affinity should be the preferred option in most cases.
+
```
Comment (lvarin) [inline docs/cloud/dbaas/web-interface.md:21]: Maybe add an image to the page.

File: docs/cloud/dbaas/web-interface.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+On the next page `Database access`:
+
```
Comment (lvarin) [inline docs/cloud/dbaas/web-interface.md:23]: same, add an image

File: docs/cloud/dbaas/web-interface.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+1. `Is public` - You need to check the public checkbox.
+2. `Allowed CIDRs` - Here you want to add your allowed IP-addresses in the format `$IP/32` if you want to allow multiple IP-addresses. You need to separate them by a comma `,`.
+3. You don't need to use the `Intialize Databases` tab. It is not recommended to create an admin user.
```
Comment (lvarin) [inline docs/cloud/dbaas/web-interface.md:26]: separate as the others and add image

**Review by lvarin — APPROVED** [review]
Looks good to us (Tristan and I)

**Pull request merged**